build.sbt

```diff
@@ -70,6 +70,9 @@ lazy val orchardWS = (project in file("orchard-ws")).
     libraryDependencies ++= Seq(
       guice,
       scalaPlusPlay
+    ),
+    dependencyOverrides ++= Seq(
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.4"
```

Comment:
is this a critical one? if not, I would prefer rely on upstream libraries to update their dependencies, since nothing in orchard uses it directly.

Reply:
Not critical - It's marked with a base score of 7.5 (high)